sql/inventory_reconciliation.sql

```diff
@@ -26,5 +25,12 @@ INSERT INTO @accounting VALUES ('ZZZ99', 1930.62)
 INSERT INTO @accounting VALUES ('xxccM', 7602.75)
 INSERT INTO @accounting VALUES ('fbr77', 17.99)
 
---TODO-CHALLENGE: Write a query to reconcile matches/differences between the inventory and accounting tables
-SELECT * FROM ...
\ No newline at end of file
+SELECT 	Inv.ItemNumber as InventoryItemNum
+	,Acc.ItemNumber as AccountingItemNum
+	,Inv.QuantityOnHand
+	,Acc.TotalInventoryValue as TotalAccountingInventoryValue
+	,Acc.TotalInventoryValue - Inv.QuantityOnHand as Difference

```

Comment:
What will this return if one of the values is null? 
Also, subtracting quantity on hand from inventory value is apples to oranges... those two numbers don't go together. The result is meaningless.